modules/apps/29-fee/keeper/escrow.go

```diff
@@ -83,7 +85,7 @@ func (k Keeper) DistributeFeeTimeout(ctx sdk.Context, refundAcc, timeoutRelayer
 	// check if there is a Fee in escrow for the given packetId
 	feeInEscrow, found := k.GetFeeInEscrow(ctx, packetID)
 	if !found {
-		return sdkerrors.Wrap(types.ErrFeeNotFound, refundAcc.String())
+		return sdkerrors.Wrapf(types.ErrFeeNotFound, "for packetID %s", packetID.String())
```

Comment:
```suggestion
		return sdkerrors.Wrapf(types.ErrFeeNotFound, "for packetID %s", packetID)
```
.String should automatically be called when formatting to string